README.md

```diff
@@ -61,116 +62,42 @@ The SDK bundle contains the following folders:
 - `include`: Contains the Sentry header file. Set the include path to this
   directory or copy the header file to your source tree so that it is available
   during the build.
-- `gen_*`: Contains generated projects and build files for the supported
-  platforms. See the next section for more information.
-- `premake`: Contains build files that allow to generate new projects and
-  customize build parameters. When building for one of the supported platforms,
-  prefer to use build files from the `gen_*` folders, instead.
 - `src`: Sources of the Sentry SDK required for building.
 
-### Platform Support
-
-The Sentry SDK comes with pregenerated build files for the following platforms:
```

Comment:
It will probably still be a good idea to list the officially supported platforms somewhere.